aspnetcore/security/authentication/sociallogins/sample/Startup.cs

```diff
@@ -87,6 +87,24 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                 AppSecret = Configuration["Authentication:Facebook:AppSecret"]
             });
 
+            app.UseTwitterAuthentication(new TwitterOptions()
```

Comment:
Maybe comment out all of the providers so the user doesn't have to set up all of them before they run? They just un-comment the provider they want.